rewrite-java/src/main/java/org/openrewrite/java/format/TabsAndIndentsVisitor.java

```diff
@@ -367,7 +367,7 @@ public Space visitSpace(Space space, Space.Location loc, P p) {
                 JContainer.build(before, js, container.getMarkers());
     }
 
-    private Space indentTo(Space space, int column) {
+    private Space indentTo(Space space, int column, boolean endOfBlock) {
```

Comment:
There are a couple places that have unique conditions to support formatting:

1. The compilation unit prefix (still not supported).
2. Comments before the end of blocks.

Comments that happen before the end of a block or in an empty block exist on the closing brace.
The expected column of the space is 1 style.getIndentSize() less then it should be for each of the comments in the list.
The space doesn't seem to have any context available for it's location when indentTo is called, so I added a check for endOfBlock.

I don't really like how it's implemented, but would like to solicit feedback before adjusting anything else.

Reply:
Tracey and I chatted about this, suggested he pass the `Space.Location` into `indentTo()`, this will allow contextual formatting of comments.